lib/index.js

```diff
@@ -62,7 +76,9 @@
  *   with `unwrapDisallowed` the element itself is replaced by its children.
  * @property {UrlTransform | null | undefined} [urlTransform]
  *   Change URLs (default: `defaultUrlTransform`)
- *
+ */
+
+/**
```

Comment:
I agree with these changes, but it would be good to have different commits for things of varying importance. This commit/PR is very important, many humans will look at the change, and see this mix of important/trivial.
Not blocking, but like: feel free to commit such refactors things directly to main (`Refactor code-style`), and use a PR for things that need eyes on them!

Reply:
I agree. It’s why I created #878 first and based this change on that one. I’ll merge it.